config/src/config.rs

```diff
@@ -369,23 +369,8 @@ impl Config {
             Environment::Testnet => &Testnet,
         };
 
-        let consensus_constants = match config.environment {
-            // When in mainnet, ignore the [consensus_constants] section of the configuration
-            Environment::Mainnet => {
-                let consensus_constants_no_changes = PartialConsensusConstants::default();
-                // Warn the user if the config file contains a non-empty [consensus_constant] section
-                if config.consensus_constants != consensus_constants_no_changes {
-                    log::warn!(
-                        "Consensus constants in the configuration are ignored when running mainnet"
-                    );
```

Comment:
It would be nice to keep this warning, but removing "when running mainnet"

Reply:
Not possible because logging is enabled after parsing the config, so any logs here will not be displayed.